src/App.js

```diff
@@ -1,18 +1,136 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import shortid from 'shortid';
+
+// import Component
+import Header from './Components/Header';
+import TableHeads from './Components/TableHeads';
+import TableItems from './Components/TableItems';
+import AddForm from './Components/AddForm';
 
 class App extends Component {
+
+  constructor() {
+    super();
+    this.state = {
+      tableHeads: [],
+      tableItems: []
+    }
+  }
+
+  // mount data before render
+  componentWillMount() {
+    this.setState({
```

Comment:
Next time you can define your fake data on other file
```js
// fakeData.js
export default {
  heads: [
   //omit..
  ],
  items: [
  //omit..
  ]
}
```
import shortid from 'shortid';
```js
import { heads, items } from './xxx/fakeData'
//omit..
this.setState({
  tableHeads: heads,
  tableItems: items
})

```